fetch.bs

```diff
@@ -7906,14 +7906,6 @@ method steps are:
  <li><p>If <var>request</var>'s <a for=request>body</a> is not null and is
  <a for=ReadableStream>readable</a>, then <a for=ReadableStream>cancel</a> <var>request</var>'s
  <a for=request>body</a> with <var>error</var>.
-
- <li><p>If <var>responseObject</var> is null, then return.
```

Comment:
What about if we're fetching from cache, not network? What about if we're fetching a blob?

Also, if this did go through, we'd delete the _responseObject_ argument.

Reply:
This was a good comment. It resulted in #1505, #1506, #1507, and #1508 thus far. I suspect I should reinstate this, but we might still want some follow-up to also create a stream for HTTP cache cases.